jetty-core/jetty-http/src/main/resources/org/eclipse/jetty/http/mime.properties

```diff
@@ -101,7 +101,7 @@ ods=application/vnd.oasis.opendocument.spreadsheet
 odt=application/vnd.oasis.opendocument.text
 ogg=application/ogg
 otc=application/vnd.oasis.opendocument.chart-template
-otf=application/vnd.oasis.opendocument.formula-template
```

Comment:
This is still the extension for OpenDocument formula templates.

* Page 698 in https://groups.oasis-open.org/higherlogic/ws/public/download/12572/OpenDocument-v1.0-os.pdf/latest

And is properly registered with the IANA

* https://www.iana.org/assignments/media-types/application/vnd.oasis.opendocument.formula-template

Reply:
@joakime I suggest you improve your reading comprehension. In the first sentence of this pull request, I write:

> A `.otf` file on a web server is more likely to be an OpenType Font than an OpenDocument Formula Template.